schema.graphql

```diff
@@ -176,6 +176,23 @@ type NominationSpend @entity(immutable: true) {
   transactionHash: Bytes!
 }
 
+enum ActivityType {
+  Vote
+  Stake
+  Ratify
+  Reject
+  Create
+}
+
+type Activity @entity {
+  id: ID!
+  user: User!
+  activityType: ActivityType!
+  timestamp: BigInt!
+  transactionHash: Bytes!
+  proposal: Proposal
```

Comment:
For ratification and rejection vote breakdown, do you think i should add an optional property of type MostPopularProposalVoteBreakdown
it can be assigned in ratification and rejection activities